mpack-instance-manager/src/main/python/instance_manager/instance_manager.py

```diff
@@ -686,5 +689,12 @@ def build_json_output(self, output_conf_dir, output_log_dir, output_run_dir, out
     if output_run_dir:
       result['run_dir'] = os.path.join(self.component_path, RUN_DIRECTORY_NAME)
     if output_path:
-      result['path'] = self.path_exec
+      result['mpack_path'] = self.path_exec
```

Comment:
This needs a change in `ambari-common`, too.

```
  File "/usr/lib/ambari-agent/lib/resource_management/libraries/functions/mpack_manager_helper.py", line 96, in get_component_home_path
    component_instance_name=component_instance_name)
  File "/usr/lib/ambari-agent/lib/resource_management/libraries/functions/mpack_manager_helper.py", line 83, in get_component_target_path
    COMPONENT_INSTANCES_PLURAL_KEY_NAME][component_instance_name][PATH_KEY_NAME]
KeyError: 'path'
```

Reply:
Not sure why we need change in mpack_manager_helper.py, the change in instance_manager.py is to list_instance, in mpack_manager_helper.py, they are to get_component_home_path and  get_component_target_path.

Reply:
Because `get_component_target_path` calls `list_instances`?

https://github.com/apache/ambari/blob/22c2e5b3d57a12ce23d8a9eee4b95370972b2f9e/ambari-common/src/main/python/resource_management/libraries/functions/mpack_manager_helper.py#L72-L80

Reply:
You are right. Thanks.